cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -106,7 +107,9 @@ public CruiseControlApiImpl(Vertx vertx) {
                                 // If the response contains a "progress" key then the rebalance proposal has not yet completed processing
                                 ccResponse.setProposalIsStillCalculating(true);
                             } else {
-                                result.fail(new CruiseControlRestException("202 Status code did not contain progress key. Response was: " +
+                                result.fail(new CruiseControlRestException(
+                                        "Error for request: " + host + ":" + port + path +
+                                        ". 202 Status code did not contain progress key. Server returned: " +
```

Comment:
what about not hard coding "202" but using `response.statusCode()` instead? If for any reason this status code will change, you have to change it in the if condition and in this error message, while using `response.statusCode()` in the error message you have just to fix one place in the if statement.

Reply:
This is inside a if branch where `if (response.statusCode() == 202)` so it can't be anything else. On line 140 there is the generic status code catch.

Reply:
Yes I know it can't be anything else but I meant this ...
Imagine that CC change and the scenario you have to check moves from 202 to 204 (just as an example).
Then you change the if statement from `response.statusCode() == 202` to `response.statusCode() == 204` but you forget to change the 202 in the error message.
The raised exception will contain 202 instead of the new 204.

Reply:
Ok in that case yes. Maybe a switch statement would have been better here.